src/Nancy/Routing/RouteDescription.cs

```diff
@@ -73,5 +77,14 @@ public RouteDescription(string name, string method, string path, Func<NancyConte
         /// </summary>
         /// <value>An <see cref="IEnumerable{T}"/>, containing the segments for the route.</value>
         public IEnumerable<string> Segments { get; set; }
+
+        private string DebuggerDisplay
+        {
+            get
+            {
+                var fields = new[] {this.Name, this.Method, this.Path}.Where(x => !String.IsNullOrEmpty(x)).ToArray();
+                return String.Join(" - ", fields);

```

Comment:
Same thing as above, do you think it's better to include the property names? 
So this would look like:
Name=RouteName Method=METHOD Path=/whatever